scripts/desktop-wallet-ci.Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Installed dependencies:
+#   - libudev-dev
+#   - libusb-1.0-0-dev
+#   - wasm-pack
+#   - yarn
+#   - node:${NODE_VERSION}
+
+ARG BASE_VERSION
+FROM concordium/base:${BASE_VERSION}
+
+RUN apt-get update && apt-get install -y \
+    libudev-dev \
+    libusb-1.0-0-dev \
+    rpm \
+    awscli
+    
+RUN curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
```

Comment:
I do not really like that we do not pin it, but since I assume this is the same as what was being done on jenkins I am fine with it